projects/hslayers/src/components/styles/styler.service.ts

```diff
@@ -411,11 +412,64 @@ export class HsStylerService {
   }
 
   async addRule(
-    kind: 'Simple' | 'ByScale' | 'ByFilter' | 'ByFilterAndScale' | 'Cluster',
-    app: string
+    kind:
+      | 'Simple'
+      | 'ByScale'
+      | 'ByFilter'
+      | 'ByFilterAndScale'
+      | 'Cluster'
+      | 'ColorMap',
+    app: string,
+    colorMapName?: string
   ): Promise<void> {
     const appRef = this.get(app);
     switch (kind) {
+      case 'ColorMap':
+        const colors = colormap({
+          colormap: colorMapName,
+          nshades: 11,
+          format: 'hex',
+          alpha: 1,
+        });
+        const attr = prompt(
```

Comment:
Will need this as translation i suppose.

Reply:
Yes, added that